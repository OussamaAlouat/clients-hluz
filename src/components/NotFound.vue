<template>
  <div>
    <v-alert type="info" icon="mdi-home-alert">
      There are no clients for that CUT
    </v-alert>
  </div>
</template>

<script>
export default {
  name: 'NotFound',
};
</script>

<style lang="scss">
</style>
