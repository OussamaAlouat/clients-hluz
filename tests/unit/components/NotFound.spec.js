import { shallowMount } from '@vue/test-utils';
import NotFound from '@/components/NotFound.vue';
import Vuetify from 'vuetify';

describe('NotFound.vue', () => {
  it('Component render', () => {
    const wrapper = shallowMount(NotFound, {
      Vuetify,
    });
    expect(wrapper).toMatchSnapshot();
  });
});
