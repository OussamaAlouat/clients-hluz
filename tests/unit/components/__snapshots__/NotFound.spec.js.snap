// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue Component render 1`] = `
<div>
  <v-alert-stub tag="div" value="true" closelabel="$vuetify.close" closeicon="$cancel" icon="mdi-home-alert" type="info">
    There are no clients for that CUT
  </v-alert-stub>
</div>
`;
